polyglot.js

```diff
@@ -185,13 +193,15 @@ class PolyGlot {
                     message.polyglot_unknown = false;
                 if (!message.polyglot_force && message.polyglot_unknown) {
                     let content = html.find(".message-content")
-                    let new_content = content.text().replace(this.regexp_letters, this.randomRune)
+                    let new_content = this.generateRune(message.data.content,game.settings.get('polyglot','salt') === true ? message.data._id : lang)
                     content.text(new_content)
+                    content[0].style = this.tongues[lang] === undefined ? 'font:' + this.alphabets[this.tongues._default] : 'font:' + this.alphabets[this.tongues[lang]]
```

Comment:
You are overriding the style here, which could conflict with other modules that change the style. It's better to do `content[0].style.font = this._getFontStyle(lang);`
This also will make it cleaner than the long one line operation